shell/platform/embedder/embedder.cc

```diff
@@ -2247,6 +2247,7 @@ FlutterEngineResult FlutterEngineSendPlatformMessageResponse(
     const FlutterPlatformMessageResponseHandle* handle,
     const uint8_t* data,
     size_t data_length) {
+  // Note: This can execute on any thread.
```

Comment:
Implementation comments about the entire method go above the definition, not inside it, in Google style. (https://google.github.io/styleguide/cppguide.html#Function_Comments "Function Definitions")

Reply:
done